en/docs/InstallAndSetup/SettingUpDatabases/ChangingDefaultDatabases/changing-to-ibm-informix.md

```diff
@@ -1,283 +1,172 @@
+
+    Sample configuration is shown below:
+    
+    ``` tab="Format"
+    url = "jdbc:informix-sqli://localhost:1533/<DATABASE_NAME>;CLIENT_LOCALE=en_US.utf8;DB_LOCALE=en_us.utf8;IFX_USE_STRENC=true;"
+    username = "<USER_NAME>"
+    password = "<PASSWORD>"
+    driver = "com.informix.jdbc.IfxDriver"
+    validationQuery = "SELECT 1"
+    pool_options.<OPTION-1> = <VALUE-1>
+    pool_options.<OPTION-2> = <VALUE-2>
+    ...
     ```
-
-### Creating database tables
-
-To create the database tables, connect to the database that you created earlier and run the following scripts.
-
-1.  To create tables in the registry and user manager database ( `WSO2CARBON_DB` ), use the below script:
-
-    ``` java
-            <PRODUCT_HOME>/dbscripts/informix.sql
+    
+    ``` tab="Example"
+    [database.shared_db]
+    url = "jdbc:informix-sqli://localhost:1533/shared_db;CLIENT_LOCALE=en_US.utf8;DB_LOCALE=en_us.utf8;IFX_USE_STRENC=true;"
+    username = "regadmin"
+    password = "regadmin"
+    driver = "com.informix.jdbc.IfxDriver"
+    validationQuery = "SELECT 1"
+    pool_options.maxActive = 100
+    pool_options.maxWait = 10000
+    pool_options.validationInterval = 10000
+    
+    [database.apim_db]
+    url = "jdbc:informix-sqli://localhost:1533/apim_db;CLIENT_LOCALE=en_US.utf8;DB_LOCALE=en_us.utf8;IFX_USE_STRENC=true;"
+    username = "apimadmin"
+    password = "apimadmin"
+    driver = "com.informix.jdbc.IfxDriver"
+    validationQuery = "SELECT 1"
+    pool_options.maxActive = 50
+    pool_options.maxWait = 30000
     ```
 
-2.  Restart the server.
-
-        !!! info
-    You can create database tables automatically **when starting the product for the first time** by using the `-Dsetup` parameter as follows:
-
-    -   For Windows: `<PRODUCT_HOME>/bin/wso2server.bat -Dsetup`
-
-    -   For Linux: `<PRODUCT_HOME>/bin/wso2server.sh -Dsetup`
-
-        !!! warning
-        Deprecation of -DSetup
-    When proper Database Administrative (DBA) practices are followed, the systems (except analytics products) are not granted DDL (Data Definition) rights on the schema. Therefore, maintaining the `-DSetup` option is redundant and typically unusable. **As a result, from [January 2018 onwards](https://wso2.com/products/carbon/release-matrix/) WSO2 has deprecated the `-DSetup` option** . Note that the proper practice is for the DBA to run the DDL statements manually so that the DBA can examine and optimize any DDL statement (if necessary) based on the DBA best practices that are in place within the organization.
-
-
+    !!! info
+        For more information on other parameters that can be defined in the `<API-M_HOME>/repository/conf/datasources/master-datasources.xml` file, see [Tomcat JDBC Connection Pool](http://tomcat.apache.org/tomcat-7.0-doc/jdbc-pool.html#Tomcat_JDBC_Enhanced_Attributes).
```

Comment:
We should not mention regarding master-datasources.xml. Should only mention regarding deployment.toml. Hence, shall we modify this note?